src/main/java/at/hannibal2/skyhanni/features/inventory/HeldTimeInLore.kt

```diff
@@ -0,0 +1,38 @@
+package at.hannibal2.skyhanni.features.inventory
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.LorenzToolTipEvent
+import at.hannibal2.skyhanni.utils.ItemUtils.getInternalName
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.NEUInternalName.Companion.asInternalName
+import at.hannibal2.skyhanni.utils.SkyBlockItemModifierUtils.getBottleOfJyrreSeconds
+import at.hannibal2.skyhanni.utils.SkyBlockItemModifierUtils.getSecondsHeld
+import at.hannibal2.skyhanni.utils.TimeUtils.format
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.DurationUnit
+import kotlin.time.toDuration
+
+object HeldTimeInLore {
+    private val config get() = SkyHanniMod.feature.inventory
+
+    private val jyrreBottle by lazy { "NEW_BOTTLE_OF_JYRRE".asInternalName() }
+    private val cacaoTruffle by lazy { "DARK_CACAO_TRUFFLE".asInternalName() }
+
+    @SubscribeEvent
+    fun onTooltip(event: LorenzToolTipEvent) {
+        if (!config.timeHeldInLore) return
+        if (!LorenzUtils.inSkyBlock) return
+        if (event.toolTip.size < 10) return // Fixes conflict with NEU's sort warning
```

Comment:
can you let me know if my patch fixes the need for this as I think it should, if so ill open a pr with that and otherwise this is fine